### PR TITLE
Chemical dispensers now use plasma to cut power usage in half and guarantee max purity dispenses

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Chemical dispensers now use plasma to cut power usage in half and guarantee max purity dispenses

## Why It's Good For The Game

Discord_pinn0iUt42

I believe we should try the carrot instead of the stick here. We have tried the stick in medical before with atmos in the form of cryogenics requiring knockout gas, and it didn't work. Let's try a positive interaction between medical and chemistry, and not a negative one.

Fixes #78222 

## Changelog

:cl: Ghilker, Iamgoofball
balance: Chemical dispensers now use plasma to cut power usage in half and guarantee max purity dispenses
/:cl:

### CODE REVIEW

File: html/changelogs/archive/2017-08.yml [hunk 1]
```diff
@@ -164,7 +164,7 @@
   - balance: Lowers the chance for monkeys to become gorillas
   - rscadd: Holoparasite prompts now have a "Never For This Round" option
   Naksu:
-  - spellcheck: fixed inconsistent grammar between machines that derive from /obj/machinery/chem_dispenser
+  - spellcheck: fixed inconsistent grammar between machines that derive from /obj/machinery/atmospherics/components/unary/chem_dispenser
```
Comment (mc-oofert) [inline html/changelogs/archive/2017-08.yml:167]: what

**Comment by mc-oofert** [comment]
also make an updatepaths script

File: html/changelogs/archive/2017-08.yml (hunk 1 above)
Comment (Iamgoofball) [reply html/changelogs/archive/2017-08.yml:167]: @Ghilker you botched your find/replace

**Comment by Sealed101** [comment]
this is much more sensible thank you

i was writing a wall of text on the other PR and i think some of the points still apply, depending on whether maps will have integrated setups for plasma supply (including a plasma can) shift start or not
the wall of text is also not revised for the carrot here, tl;dr would be is mapping plasma in for these areas a good idea or not

i know goof didn't add any map changes outside of the repathing of the dispensers but still

<details><summary>spoilering it because i can't be assed to revise it for this PR</summary>
<p>
this is an interesting idea that has been around for a while but i don't think it will be beneficial to the overall gameplay.

### what i dislike

- chemistry already can and does cause fires and explosions, i don't see why it should have more vulnerabilities/workspace hazard rating (also common chemistry breaches will heavily negate any gas fires i'd imagine)
  - bartender does not do explosions or fires except an alcohol fire trick or two, however. though fitting an area that attracts chaotic behaviors of the Holy Bar RP with plasma gas could be funny
- there isn't any merit for jobs that work with gases to actually supply chemistry with plasma gas outside of RP value, they don't really gain anything out of it unless they need medical help. 
  - depending on the mapped-in setup for these (or lack thereof), chemistry use would vastly differ. and getting it to work would involve atmos techs that aren't guaranteed to exist on every round (or care to setup things for medbay, see cryogenics). 
    - though bartenders don't usually run out of their power supply in a typical shift, having to care for gas while serving customers (players or NPCs) seems like a huge roadblock for basically no gain
  - and failing that, chemistry/bartender would have to go either bug toxins or tide into places. 
  - this can be somewhat rectified by giving medical access to plasma canister cargo orders but that's just a medbay plasmafire waiting to happen 
-  while it's always fun to get a new sabotage opportunity, running plasma gas in a department that's almost universally doomed to be gone by the end of the shift is a very low kick in the gut for medbay.
  - this applies to the cafeteria/bar as well but not so much, it wreaking havoc is in the area description.

### what do we gain from this?

- chemistry/bartender work (as before with no changes in productivity)
- atmos techs get something to tweak outside of their department (say hi to cryogenics)
- closing a loophole in immersion, as those pesky machines magic'd power into chemicals, drinks and what have you
- funny purple gas in places where it shouldn't be, i wonder if i should make ClF3/light a cigar in it...

also i don't think abductors should care for plasma, they don't have much space on the saucer already nor can they bring a plasma canister up. best they can do is bringing plasma tanks i guess?

</p>
</details>

**Comment by dieamond13** [comment]
goof what does "guarantee max purity dispenses" mean, dispensers already dispense everything at 100%

**Comment by Iamgoofball** [comment]
@dieamond13 not all dispensers do that actually

File: html/changelogs/archive/2017-08.yml (hunk 1 above)
Comment (Ghilker) [reply html/changelogs/archive/2017-08.yml:167]: Damn i forgot to blacklist these files, my bad

**Comment by chemistrymain2** [comment]
this should also cause the machine to explode if the plasma is too hot (because this amuses me)

**Comment by lessthnthree** [comment]
Good idea, is plasma released into the environment if the machine is destroyed? It'd be nice to add some risk to reward for playing with fire (literally.)

**Comment by Iamgoofball** [comment]
alright so i checked ghilker's commit and it removes every bar dispenser in the game

**Pull request closed without merging**